### PR TITLE
Clean unused imports

### PR DESCRIPTION
**Pull request by obycode**
### Description

```
use regex::internal::Exec;
```
causes an error with newer versions of the `regex` crate, and it is not
used anyway, so remove it.

Also re-format using the recommended:
```
rustfmt clarity/src/vm/costs/mod.rs --config group_imports=StdExternalCrate
```

### CODE REVIEW

**Review by kantai — APPROVED** [review]
LGTM -- note that `Exec` was removed in `master` as well.

**Comment by obycode** [comment]
> note that `Exec` was removed in `master` as well.

Oh, good. Then maybe we should not merge this and instead merge master back to develop?

**Comment by kantai** [comment]
Yeah, `master` should be merged to `develop` -- I was going to open a PR to do so, but am waiting until the github action fixes are merged.

**Comment by obycode** [comment]
Ok, cool. I'll close this one then.

**Pull request closed without merging**
